ui/apps/platform/src/Containers/Vulnerabilities/NodeCves/Overview/CVEsTable.tsx

```diff
@@ -1,48 +1,230 @@
 import React from 'react';
-import { Bullseye, Spinner } from '@patternfly/react-core';
+import { Link } from 'react-router-dom';
+import { Text } from '@patternfly/react-core';
+import { ExpandableRowContent, Table, Tbody, Td, Th, Thead, Tr } from '@patternfly/react-table';
+import { gql, useQuery } from '@apollo/client';
 
-import TableErrorComponent from 'Components/PatternFly/TableErrorComponent';
+import {
+    TbodyLoading,
+    TbodyError,
+    TbodyEmpty,
+    TbodyFilteredEmpty,
+} from 'Components/TableStateTemplates';
+import useSet from 'hooks/useSet';
 import useURLPagination from 'hooks/useURLPagination';
+import { getTableUIState } from 'utils/getTableUIState';
 
+import DateDistance from '../../components/DateDistance';
+import { getNodeEntityPagePath, getRegexScopedQueryString } from '../../utils/searchUtils';
+import TooltipTh from '../../WorkloadCves/components/TooltipTh';
+import { DynamicColumnIcon } from '../../components/DynamicIcon';
```

Comment:
These 4 imports in particular look like they could be lifted up to the app-wide Components folder.

Reply:
I have a "move stuff around" PR coming soon that addresses all of these in the recent batch of PRs.